newsfragments/3200.feature.rst

```diff
@@ -0,0 +1 @@
+Return iterable of `ABIFunction`s from the `BaseContractFunctions` iterator.
```

Comment:
We should label this as `3200.breaking.rst` since it breaks the current behavior.